scripts/benchmark_k.py

```diff
@@ -157,7 +174,7 @@ def handleArgs(argv):
     'min_epochs': 25,
     'max_epochs': max_epochs,
     'auto_lr': True,
-    'max_lr': 0.0001,
+    'max_lr': 0.001,
```

Comment:
I see that here you set LR based on the model type. Does this get overriden ever by the dataset? I remember last time something about a zeisel learning rate

Reply:
Ah good catch, I think this was a mistake. This parameter may be data set dependent, but I just ran a quick benchmark between these 2 values and it doesn't seem to make much of a difference. I will just leave it at 0.0001.